res/TensorFlowLiteRecipes/L2Normalize_U8_000/test.recipe

```diff
@@ -0,0 +1,22 @@
+operand {
+  name: "ifm1"
+  type: UINT8
+  shape { dim: 1 dim: 4 dim: 4 dim: 3 }
+  quant { min: 0 max: 255 scale: 0.0078125 zero_point: 128}
```

Comment:
It would be better to make `scale` 1.

In `circle-quantizer` (and others doing affine quantization), `scale` is calculated as below.

scale = (max - min) / (qmax - qmin)

Since this recipe is for u8 quantization, qmax = 255 and qmin = 0. So, scale would be `(max - min) / 255`.

Reply:
I'm sorry, I made a mistake. I should fix min & max
Scale should be fixed to 0.0078125 for 2 reasons:
    1. According to guide([Tensorflow Quantization](https://www.tensorflow.org/lite/performance/quantization_spec)),
        L2Normalization has scale of (1.0 / 128.0)
    2. If I set scale to 1.0, ./nncc build fails and outputs error message: `testDataGenerator: /home/bearpaek/data/git/opensource/ONE/compiler/luci-interpreter/src/kernels/L2Normalize.cpp:42: virtual void luci_interpreter::kernels::L2Normalize::configure(): Assertion `output()->scale() == (1. / 128.)' failed.`

So I think it is reasonable to keep scale as 0.0078125, and change max to 2.

I will commit again after changing max.

Thank you for your kind guidance, @jinevening .

Reply:
Nice finding. We should use the scale 0.0078125, as you mentioned. The exact value of max is 255 * 0.0078125 = 1.9921875, but it would be ok to just use 2 (min/max are typically not important).

TensorFlow restricts the scale and zero point of several Ops.

log_softmax: {16.0/256.0, 127}  ({scale, zp})
logistic: {1 / 256.0, -128}
l2_normalization: {1 / 128.0, 0}
softmax: {1 / 256.0, -128}
tanh: {1 / 128.0, 0}

If you are going to make a recipe for one of those operators, please use the specified values as you did in this PR. Thanks.

Reply:
Thank you for comment @jinevening ,
I will do the same for the rest of the recipes! :)